app/src/util/form-schema/index.ts

```diff
@@ -1,3 +1,46 @@
 import SectorFormSchema from "./sector-form-schema.json";
+import HIERARCHY from "./manual-input-hierarchy.json";
```

Comment:
The new json file was already loaded and imported in the my recent PR. Did you base this PR on that?